var/spack/repos/builtin/packages/py-neurodamus/package.py

```diff
@@ -23,3 +24,13 @@ class PyNeurodamus(PythonPackage):
     depends_on('py-lazy-property', type='run')
     depends_on('py-docopt',        type='run')
     depends_on('py-six',           type='run')
+
+    @run_after('install')
+    def install_scripts(self):
+        mkdirp(self.prefix.share)
+        for script in ('init.py', '_debug.py'):
+            copy(script, self.prefix.share)
+
+    def setup_environment(self, spack_env, run_env):
+        PythonPackage.setup_environment(self, spack_env, run_env)
```

Comment:
Good call, but a heads up: with the new deployment, `setup_environment` is deprecated. Packages upstream may migrate to `setup_{run,build}_environment`, so the parent method is not guaranteed to exist.

Nothing urgent here, just a note to myself and FYI.